src/server/utils/config.ts

```diff
@@ -2,8 +2,6 @@ import type { SessionConfig } from 'h3';
 
 import debug from 'debug';
 
-export const PORT = process.env.PORT || '51821';
```

Comment:
Are you sure to remove `PORT` ?

Reply:
yes because its not used anywhere. h3 (nuxt) needs `PORT` as env var as you don't start the server yourself. `WEBUI_HOST` has to be migrated to `HOST` to work